syndesis/plugins/org.fusesource.ide.syndesis.extension.core/src/org/fusesource/ide/syndesis/extensions/core/util/SyndesisExtensionsUtil.java

```diff
@@ -42,4 +52,87 @@ public static boolean isValidSyndesisExtensionVersion(String version) {
 		}
 		return valid;
 	}
+	
+	/**
+	 * retrieves important version information from an online version mapping file
+	 * 
+	 * @param urlString
+	 * @return
+	 * @throws IOException
+	 */
+	public static IgniteVersionInfoModel getIgniteVersionModel(String urlString) {
+		IgniteVersionInfoModel model = new IgniteVersionInfoModel();
+		Properties vMapping = new Properties();
+		
+		try {
+			URL url = new URL(urlString);
+			vMapping.load(url.openStream());
+		} catch (IOException ex) {
+			// we ignore load errors
+		}
+				
+		model.setCamelVersion(vMapping.getProperty(KEY_CAMEL_VERSION, "2.20.1"));
+		model.setSpringBootVersion(vMapping.getProperty(KEY_SPRING_BOOT_VERSION, "1.5.8.RELEASE"));
+		model.setSyndesisVersion(vMapping.getProperty(KEY_SYNDESIS_VERSION, "1.2.2"));
```

Comment:
why 1.2.2? I thought that it was 1.2.0. Do you have a pointer for the 1.2.2 artefact?

Reply:
because that will be most likely the minimum TP3 version number regarding Roland

Reply:
https://mvnrepository.com/artifact/io.syndesis/syndesis-rest-parent/1.2.2 is available here.
There is also a 1.2.3 https://mvnrepository.com/artifact/io.syndesis/syndesis-rest-parent/1.2.3 but maybe built after Tech Preview 3. Where can we found the list of versions for Tech preview 3?

Reply:
1.2.3 is the TP3 version. we need to adapt the version file.